Cargo.toml

```diff
@@ -50,10 +50,14 @@ secp256k1 = { version = "0.20.0", features = [ "recovery", "rand-std" ] }
 bincode = "1.3.1"
 # We need to pin ryu (transitive dep from serde_json) to stay compatible with Rust 1.22.0
 ryu = "<1.0.5"
+base64-compat = "1.0.0"
```

Comment:
Why is this here? I don't see it used anywhere in the PR?

Reply:
The examples/psbt.rs file needed it to use base64::decode.

Reply:
Oh nice, I didn't click that `base64-compat` had the lib name set to `base64`. Thanks